lib/membrane/aac/parser/adts.ex

```diff
@@ -122,12 +127,13 @@ defmodule Membrane.AAC.Parser.Helper do
     freq_index = stream_format.sample_rate |> AAC.sample_rate_to_sampling_frequency_id()
     channel_config = stream_format.channels |> AAC.channels_to_channel_config_id()
     profile = AAC.profile_to_aot_id(stream_format.profile) - 1
+    mpeg_version_id = stream_format.mpeg_version |> AAC.mpeg_version_to_mpeg_version_id()
 
     header = <<
       # sync
       0xFFF::12,
-      # id
-      0::1,
+      # mpeg version id
```

Comment:
that comment does not provide much information ;) I would either remove it, or provide a description of the values (0 => MPEG-4, 1=> MPEG-2)

Reply:
it follows the convention of the whole header: 
```
header = <<
      # sync
      0xFFF::12,
      # mpeg version id
      mpeg_version_id::1,
      # layer
      0::2,
      # protection_absent
      1::1,
      # profile
      profile::2,
      # sampling frequency index
      freq_index::4,
      # private_bit
      0::1,
      # channel configuration
      channel_config::3,
      # original_copy
      0::1,
      # home
      0::1,
      # copyright identification bit
      0::1,
      # copyright identification start
      0::1,
      # aac frame length
      frame_length::13,
      # adts buffer fullness (signalling VBR - most decoders don't care anyway)
      0x7FF::11,
      # number of raw data blocks in frame - 1
      0::2
    >>
```